src/main/java/com/uid2/admin/vertx/service/EncryptionKeyService.java

```diff
@@ -316,22 +328,24 @@ private RotationResult rotateKeys(Predicate<Integer> siteSelector, Duration minA
             return result;
         }
 
-        result.rotatedKeys = addSiteKeys(siteIds, activatesIn, expiresAfter);
+        result.rotatedKeys = addSiteKeys(siteIds, activatesIn, expiresAfter, true);
 
         return result;
     }
 
-    private List<EncryptionKey> addSiteKeys(Iterable<Integer> siteIds, Duration activatesIn, Duration expiresAfter)
+    private List<EncryptionKey> addSiteKeys(Iterable<Integer> siteIds, Duration activatesIn, Duration expiresAfter, boolean duringRotation)
```

Comment:
NP it's a convention to name boolean vars with is/are e.g. isDuringRotation so it's obvious which way the boolean goes.

Reply:
Changed!